common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -870,6 +870,15 @@ def test_course_home(self):
         self.course_home_page.outline.resume_course_from_outline()
         self.assertTrue(self.courseware_page.nav.is_on_section('Test Section 2', 'Test Subsection 3'))
 
+        # Test that events are emitted when outline sections are clicked
```

Comment:
@bjacobel @andy-armstrong: Is this test doing anything different from the JS test, or could we just kill this redundancy in bokchoy?